app/mailers/welcome_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class WelcomeMailer < ApplicationMailer
```

Comment:
I'd rather see these live in `DeveloperMailer` and `BusinessMailer`. Can you please refactor to accommodate that?